uber/models.py

```diff
@@ -545,7 +545,7 @@ def next_badge_num(self, badge_type, old_badge_num):
 
             # Adjusts the badge number based on badges in the session
             for attendee in [m for m in chain(self.new, self.dirty) if isinstance(m, Attendee)]:
```

Comment:
Yeah.  This is a fix for the situation where an admin manually assigns a badge number that's out of range, like if Staff badges only go up to 200 but the admin types "250" in the field.  In this case we want ``next_badge_num`` to return whatever the next available number is within the valid range, rather than returning 251.